.cargo/config.toml

```diff
@@ -1,2 +1,5 @@
 [build]
 target = "wasm32-unknown-unknown"
+
+[target.wasm32-unknown-unknown]
+rustflags = ['--cfg', 'getrandom_backend="wasm_js"']
```

Comment:
Smart to put that here 👌.

Reply:
I can't claim too much credit. I just read the `getrandom` docs and did what it told me...